itou/www/siae_evaluations_views/views.py

```diff
@@ -72,3 +74,51 @@ def siae_job_applications_list(request, template_name="siae_evaluations/siae_job
         "back_url": back_url,
     }
     return render(request, template_name, context)
+
+
+@login_required
+def siae_select_criterias(
```

Comment:
peut etre un transaction atomic si plusieurs critères sont modifiés ?

Reply:
oui! au niveau de `if request.method == "POST" and form_administrative_criteria.is_valid()`